Symfony/CS/Fixer.php

```diff
@@ -139,18 +140,26 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
             $config->getRules()
         );
 
-        foreach ($config->getFinder() as $file) {
-            if ($file->isDir() || $file->isLink()) {
-                continue;
-            }
+        $iterator = $config->getFinder();
+        if ($iterator instanceof \IteratorAggregate) {
+            $iterator = $iterator->getIterator();
+        }
+        /* @var \Iterator $iterator */
+        $iterator->rewind();
+
+        $nextFile = $this->getNextFile($iterator, $fileCacheManager);
 
-            $this->stopwatch->start($this->getFileRelativePathname($file));
+        while ($file = $nextFile) {
+            $iterator->next();
+            $nextFile = $this->getNextFile($iterator, $fileCacheManager);
 
-            if ($fixInfo = $this->fixFile($file, $fixers, $dryRun, $diff, $fileCacheManager)) {
-                $changed[$this->getFileRelativePathname($file)] = $fixInfo;
+            $this->stopwatch->start($this->getFileRelativePathname($file['file']));
```

Comment:
I'm not sure where to put this line?
Before this pr the time started before checking cache file and linting.
Now it starts before waiting for linting process. So checking cache file and starting the linting process are not captured by the stopwatch. (Cached files are not captured at all.)